js/holidays.js

```diff
@@ -353,6 +355,21 @@ function getMonthDays( year ) {
 	return ndays;
 }
 
+/**
+ * Returns the holiday only if it's applicable in the current region
+ *
+ * @param {array} holidayRegions Regions the holiday is observed in
+ * @param {object} holiday Holiday definition
+ * @returns Array (zero or one) of holiday definitions applicable
+ */
+function inRegions ( holidayRegions, holiday ) {
```

Comment:
Would you mind adjusting the spacing in this block? Sorry if I adopt an unusual style, but the extra spacing really improves the code readability for me.
Could also go without the curly brackets in the if-else statements, but it's your call.
```
function inRegions( holidayRegions, holiday ) {
	if ( holidayRegions.includes( region ) ) {
		return [ holiday ];
	}
	else {
		return [];
	}
}
```

Reply:
Reformatted, sorry for slipping.

I'll leave the curly brackets in [just](https://www.imperialviolet.org/2014/02/22/applebug.html) in [case](https://www.visualmicro.com/page/User-Guide.aspx?doc=Curly-Braces-Ifs.html).

Reply:
Thanks! I know it's a terrible practice, but old habits die hard.. 😅